tweepy/api.py

```diff
@@ -412,9 +412,9 @@ def direct_messages(self):
         """
         return bind_api(
             api=self,
-            path='/direct_messages.json',
+            path='/direct_messages/events/list.json',
             payload_type='direct_message', payload_list=True,
-            allowed_param=['since_id', 'max_id', 'count', 'full_text'],
+            allowed_param=['since_id', 'cursor', 'count', 'full_text'],
```

Comment:
I understand that there are only 2 parameters allowed:
count & cursor 

Source: https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/list-events

Reply:
Removed, thanks